pkg/js-search/source/IndexStrategy/ExactWordIndexStrategy.test.ts

```diff
@@ -0,0 +1,16 @@
+import { ExactWordIndexStrategy } from "./ExactWordIndexStrategy";
+describe('ExactWordIndexStrategy', function () {
+  var indexStrategy;
+  beforeEach(function () {
+    indexStrategy = new ExactWordIndexStrategy();
+  });
+  it('should not expand empty tokens', function () {
+    var expandedTokens = indexStrategy.expandToken('');
+    expect(expandedTokens.length).toEqual(0);
+  });
+  it('should not expand tokens', function () {
+    var expandedTokens = indexStrategy.expandToken('cat');
+    expect(expandedTokens.length).toEqual(1);
+    expect(expandedTokens).toContain('cat');
+  });
+});
```

Comment:
We really need prettier on pre-commit (issue already opened: #12 )

Reply:
I upgraded `prettier` to the latest version and ran it on `js-search`.